addons/im_livechat/views/res_users_views.xml

```diff
@@ -23,9 +29,14 @@
             <field name="inherit_id" ref="base.view_users_form"/>
             <field name="arch" type="xml">
                     <xpath expr="//group[@name='messaging']" position="after">
+                        <field name="has_access_livechat" readonly="1" invisible="1"/>
```

Comment:
Isn't `readonly="1"` the default, especially if `readonly=True` is set in python?
Also it seems not useful anyway on invisible field?